### PR TITLE
Implement sorter_sublime

### PR DESCRIPTION
**Pull request by wh11e7rue**
I have implemented a sorter like Sublime Text's fuzzy matching based on [this article](https://blog.forrestthewoods.com/reverse-engineering-sublime-text-s-fuzzy-match-4cffeed33fdb) and [its implementation](https://github.com/forrestthewoods/lib_fts/blob/master/code/fts_fuzzy_match.js#L33).

I'm using it with init.vim like this:
```vim
  call denite#custom#source('buffer,file_rec,file_mru', 'sorters', ['sorter_sublime'])
```

sorter_sublime:

sorter_rank:

### CODE REVIEW

File: rplugin/python3/denite/filter/sorter_sublime.py [hunk 1]
```diff
@@ -0,0 +1,155 @@
+from .base import Base
```
Comment (Shougo) [inline rplugin/python3/denite/filter/sorter_sublime.py]: Please add the LICENSE.

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,2 @@
+[*.py]
+indent_size = 4
```
Comment (Shougo) [inline .editorconfig]: It should be added?

File: .gitignore [hunk 3]
```diff
@@ -1,3 +1,33 @@
...
+.VolumeIcon.icns
+
+# Directories potentially created on remote AFP share
+.AppleDB
+.AppleDesktop
+Network Trash Folder
+Temporary Items
+.apdisk
+
+
```
Comment (Shougo) [inline .gitignore]: What is this?
Why this is added?

**Comment by Shougo** [comment]
This is the good feature.
But it includes unrelated changes.
Please explain to me.

**Comment by wh11e7rue** [comment]
Sorry...
I have deleted that commit

**Comment by Shougo** [comment]
LGTM

**Pull request merged**